desktop/core/src/desktop/js/catalog/dataCatalogEntry.js

```diff
@@ -812,13 +812,13 @@ class DataCatalogEntry {
    */
   getResolvedComment() {
     const self = this;
+    if (self.definition && self.definition.comment) {
+      return self.definition.comment;
+    }
     // TODO: Move to connector attributes
     if (self.navigatorMeta && (self.getDialect() === 'hive' || self.getDialect() === 'impala')) {
```

Comment:
Add && (self.navigatorMeta.description || self.navigatormeta.originalDescription) here instead, as nav comment should have precedence

Reply:
Done , Changes done as requested.